.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] According to the requirements, you should create `test.md` file with the text "This file should be ignored by git". then you should ignore this file by git by adding it iside `.gitignore` 💯

![image](https://user-images.githubusercontent.com/45231276/198201074-c7391410-1feb-4fd5-aba4-9aa321a1a126.png)